Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1636,7 +1631,21 @@ extension BrowserViewController: TabManagerDelegate {
             ReaderModeHandlers.readerModeCache = readerModeCache
 
             scrollController.browser = selected
+
+            // before we add a new webView, ensure that there aren't too many subviews already in the webViewContainer
+            if webViewContainer.subviews.count > 20,
+                let bottomWebView = ( webViewContainer.subviews.find {
```

Comment:
But if `subviews` is an ordered array (which it is, not a set), then `for x in self` inside `find` would iterate in order 0...n where 0 is the first element added, i.e. at the bottom of the stack. You way would find the last item added to the stack which would be the most recently added item, no?